roles/create_tables/tasks/create_db.yaml

```diff
@@ -0,0 +1,8 @@
+---
+- name: create database {{ hoh_db }} 
+  postgresql_db:
+    name: "{{ hoh_db }}"
+    login_host: "{{ db_login_host }}"
+    login_user: "{{ db_login_user }}"
+    login_password: "{{ db_login_password }}"
+    ssl_mode: "{{ db_ssl_mode }}"
```

Comment:
do we need this? is it supposed that the database should be created by end-user?